server/cmwell-cons/src/main/scala/Deployment.scala

```diff
@@ -475,9 +476,9 @@ case class BgProps(h: Host)
   override val componentName: String = "cmwell-bg"
   override val componentMappings: Map[String, Int] = Map("bg" -> 1)
 
-  override def stop(hosts: GenSeq[String]): Unit = h.stopBg(hosts)
+  override def stop(hosts: ParSeq[String]): Unit = h.stopBg(hosts)
 
-  override def start(hosts: GenSeq[String]): Unit = h.startBg(hosts)
+  override def start(hosts: Seq[String]): Unit = h.startBg(hosts.to(ParSeq))
```

Comment:
Why they're both GenSeq and you changed one to ParSeq and the other to Seq?

Reply:
The hosts is a normal Seq (ES need the ordering). StartBg method takes ParSeq.